<launch>
	<param name="ipmaster" value="127.0.0.1" type="str" />
	<param name="coordport" value="68" type="str" />
	<param name="mapport" value="67" type="str" />
	<param name="mappath" value="68" type="str" />
	<param name="midiconfigpath" value="/home/gabsen/midiconfig.json" type="str" />
	<param name="waypointpath" value="/home/gabsen/wp.json" type="str" />
	<param name="channel" value="0" type="int" />
	<param name="midiport" value="3" type="int" />
	<param name="Robo_IP" value="42.0.1.2" type="str" />
	<param name="json_midiconfig_PORT" value="66" type="int" />
	<param name="json_WP_PORT" value="65" type="int" />
</launch>
